site/content/how-to/traffic-management/tls-passthrough.md

```diff
@@ -0,0 +1,231 @@
+EOF
+```
+
+This will create the **secure-app** service and a deployment. The secure app is configured to serve HTTPS traffic on port 8443 for the host app.example.com. For TLS termination, a self-signed TLS certificate, with the common name `app.example.com`, and key are used. The app responds to clients HTTPS requests with a simple text response "hello from pod $POD_HOSTNAME".
+
+Run the following command to verify the resources were created:
+
+```shell
+kubectl get pods,svc
+```
+
+The output should include the **secure-app** pod and the **secure-app** service:
+
+```text
+NAME                              READY   STATUS      RESTARTS   AGE
+pod/secure-app-575785644-kzqf6    1/1     Running     0          12s
+
+NAME                  TYPE        CLUSTER-IP        EXTERNAL-IP   PORT(S)    AGE
+service/secure-app    ClusterIP   192.168.194.152   <none>        8443/TCP   12s
+```
+
+Create a gateway. This will create a TLS listener with the hostname `*.example.com` and a TLS mode of passthrough. Copy and paste this into your terminal.
```

Comment:
```suggestion
Create a Gateway. This will create a TLS listener with the hostname `*.example.com` and passthrough TLS mode. Copy and paste this into your terminal.
```